README.md

```diff
@@ -16,6 +16,17 @@ To start your Phoenix app:
 
 Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
 
+## Tests
+
+To run the tests, first install and setup Podman:
+  1. `brew install podman`
```

Comment:
question: I missed this until now, but we've recommended `colima` as the Docker replacement for TID: https://github.com/mbta/technology-docs/blob/main/rfcs/accepted/0010-docker-desktop-replacement.md

Is there a reason that `colima` + docker compose doesn't work here? Should we be updating the RFC?

Reply:
I'll try it with Colima; Podman is just what came to me off the top of my head as the FOSS Docker thing.